src/test/kotlin/calculator/CalculatorKoTest.kt

```diff
@@ -0,0 +1,67 @@
+package calculator
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class CalculatorKoTest : StringSpec({
+    "더하기" {
+        val calculator = Calculator()
+
+        calculator.plus(2, 1) shouldBe 3
+    }
+
+    "빼기" {
+        val calculator = Calculator()
+
+        calculator.minus(2, 1) shouldBe 1
+    }
+
+    "곱하기" {
+        val calculator = Calculator()
+
+        calculator.multiply(2, 3) shouldBe 6
+    }
+
+    "나누기" {
+        val calculator = Calculator()
+
+        calculator.division(8, 2) shouldBe 4
+    }
+
+    "사칙연산 기호가 아닌 경우 에러가 발생한다" {
+        val calculator = Calculator()
+
+        val exception = shouldThrow<IllegalArgumentException> {
+            calculator.calculate("1 , 1")
+        }
+
+        exception.message shouldBe "잘못된 사칙연산 기호입니다."
+    }
+
+    "입력값이 빈 공백일 경우 에러가 발생한다" {
+        val calculator = Calculator()
+
+        shouldThrow<IllegalArgumentException> {
+            calculator.calculate("")
+        }.message shouldBe "잘못된 입력값입니다."
+
+        shouldThrow<IllegalArgumentException> {
+            calculator.calculate(" ")
+        }.message shouldBe "잘못된 입력값입니다."
+
+        shouldThrow<IllegalArgumentException> {
+            calculator.calculate("\t")
+        }.message shouldBe "잘못된 입력값입니다."
+
+        shouldThrow<IllegalArgumentException> {
+            calculator.calculate("\n")
+        }.message shouldBe "잘못된 입력값입니다."
```

Comment:
```suggestion
        listOf("", " ", "\t", "\n").forAll { expression ->
            shouldThrow<IllegalArgumentException> {
                calculator.calculate(expression)
            }.message shouldBe "잘못된 입력값입니다."
        }
```
값을 list화 하면 좀 더 간결하게 테스트 코드를 작성하실 수 있어요.